cScripts/CavFnc/functions/players/fn_setPostInitPlayerSettings.sqf

```diff
@@ -141,3 +141,11 @@ if !(weaponLowered _player) then {_player action ["WeaponOnBack", _player]};
 if (isNil {_unit getVariable QEGVAR(Player,Unit)}) then {
     [formatText["%1 have no unit variable defined", _player], "LoadoutPostInit", true] call FUNC(logWarning);
 };
+
+//Server metrics
+_adminState = call BIS_fnc_admin;
+if (_adminState != 0) then {
```

Comment:
```suggestion
if ((call BIS_fnc_admin) >= 2) then {
```

`_adminState` was missing private but the variable can be skipped and instead you can do a direct call in the condition